client/state/followers/selectors.js

```diff
@@ -3,21 +3,29 @@
  */
 
 import { getSerializedQuery } from 'calypso/state/followers/utils';
+
 /**
  * Returns a list of followers for the given Query.
  *
  * @param  {object} state Global state tree
  * @param  {object} query Query paramaters by which the followers were fetched
  * @returns {object}       List of followers keyed by follower id
  */
-export function getFollowersForQuery( state, query ) {
+export const getFollowersForQuery = ( state, query ) => {
+	const serializedQuery = getSerializedQuery( query );
+	return (
+		state.followers?.queries[ serializedQuery ]?.ids.map(
+			( id ) => state.followers?.items[ id ]
+		) ?? []
+	);
+};
+
+export const getIsFetchingFollowersForQuery = ( state, query ) => {
```

Comment:
Naming nit: I'd rename this to `isFetchingFollowersForQuery`; `get` feels pretty redundant in a boolean "is requesting" selector.

Reply:
fixed via c9a9322